### PR TITLE
Add test for checking daemons are removed after removing libvirtd pkgs

### PR DESCRIPTION
**Pull request by yafu-1**
Signed-off-by: Yan Fu <yafu@redhat.com>

# Format of PR title < sub-system: summary >
e.g
*virsh_migrate: Fix unsupported direct socket mode issue*

# Check lists by category
## If the PR is new cases
- [ ] Description of the cases
- [ ] Links of libvirt features, libvirt bugs or case IDs
- [ ] Test results

## If the PR is bug cases
- [ ] Bug descriptions or bug links
- [ ] Test results

## If the PR is a trivial fix
It it is the fix of typos, comments or documents, the description of PR could be skipped.

### CODE REVIEW

**Comment by yafu-1** [comment]
$ avocado run daemon.check_daemon_after_remove_pkgs --vt-type libvirt
No python imaging library installed. Screendump and Windows guest BSOD detection are disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
No python imaging library installed. PPM image conversion to JPEG disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
JOB ID     : f8da4671772e25d81a8e4ff352405b26579889dd
JOB LOG    : /root/avocado/job-results/job-2022-03-30T23.32-f8da467/job.log
 (1/2) type_specific.io-github-autotest-libvirt.daemon.check_daemon_after_remove_pkgs.legacy_daemon: CANCEL: This daemon mode is not the same as the required daemon mode. (30.25 s)
 (2/2) type_specific.io-github-autotest-libvirt.daemon.check_daemon_after_remove_pkgs.modular_daemon: PASS (223.24 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 1

File: libvirt/tests/src/daemon/check_daemon_after_remove_pkgs.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        virsh.reboot(vm)
+        if session is None:
+            session = vm.wait_for_login()
+
+        #Destroy default network, otherwise network daemon will not be removed after removed libvirt pkgs
+        cmd = "virsh net-destroy default"
+        session.cmd(cmd, ignore_all_errors=True)
+
+        cmd = "systemctl start virtlogd"
+        status, output = session.cmd_status_output(cmd)
```
Comment (Yingshun) [inline libvirt/tests/src/daemon/check_daemon_after_remove_pkgs.py:43]: I'd like to share a tip for restarting service:
runner = remote.RemoteRunner(session=session).run
service.Factory.create_service('virtlogd', run=runner).start()

**Review by Yingshun — COMMENTED** [review]
Others LGTM

File: libvirt/tests/src/daemon/check_daemon_after_remove_pkgs.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import logging
+
+from virttest import virsh
+from virttest import utils_package
+from virttest import utils_split_daemons
+
+# Using as lower capital is not the best way to do, but this is just a
+# workaround to avoid changing the entire file.
```
[the remark this answers is not in the document]
Comment (yafu-1) [reply libvirt/tests/src/daemon/check_daemon_after_remove_pkgs.py]: Updated.

File: libvirt/tests/src/daemon/check_daemon_after_remove_pkgs.py (hunk 1 above)
Comment (yafu-1) [reply libvirt/tests/src/daemon/check_daemon_after_remove_pkgs.py]: Updated. Thanks for your suggestion.

**Review by dzhengfy — APPROVED** [review]
LGTM

**Pull request merged**